### ISSUE TITLE
Fiddle Instrument

### ISSUE THREAD

**Issue by Fourniture** [musescore/MuseScore#25554]
### Your idea

Please add the Instrument "Fiddle" to the list of available instruments in the Score Wizard.

### Problem to be solved

There is no fiddle available now.
![image](https://github.com/user-attachments/assets/9b78b414-11e8-4505-8f7f-ffd49904b907)

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by DaddyLudwig** [comment]
I’m pretty sure a fiddle is just a violin played in a different music genre and style than classical violin playing, that would make sense to me aas to why fiddle isn’t on the instruments list.

**Comment by Fourniture** [comment]
True, but who wants to select a violin for a bluegrass score and then have to change the instrument name to fiddle? Also I have played next to fiddlers, and have noticed they bow near the fingerboard which results in a rather different sound.

**Comment by Fourniture** [comment]
It is not just about the name of the instrument. General Midi recognizes fiddle as a separate instrument from violin. As regards the necessity of having an instrument option vs. changing the instrument name, Musescore has piano _and_ grand piano, which seems to contradict your reason for closing this issue. While fiddle is _technically_ the same as a violin, it is not generally considered the same instrument. **That is why I again request for the instrument "Fiddle" to be added to the list of instruments in the score wizard.**

**Comment by Jojo-Schmitz** [comment]
Yes, Fiddle having a sound in MIDI (and therefore in MS Basic too) is a good reason for adding it to MuseScore's instrument list

**Comment by Jojo-Schmitz** [comment]
BTW: Erhu does use that sound, but has a different range (than violin)

**Comment by MarcSabatella** [comment]
I also agree it is at least potentially worth adding.  The various different listings we have for guitars and basses is another example where we do this already.  Also including the different transpositions for woodwinds and brass as separate instruments is of a similar nature.  And of course, it costs virtually nothing to have it there, probably in the "world music" and/or "pop/rock" categories.

**Comment by Dima-S-Jr** [comment]
I agree with what has been said.
Indeed, FIddle is available in General MIDI under the number 111 (or 110 when numbered from 0 to 127).
I also want to add that, unlike playing a regular violin, Fiddle is almost always played in a non-vibrato style. 
It can also be noted that the vast majority of Fiddle uses steel strings to make the sound closer to the genre from which the name of this instrument follows. This makes the sound more sonorous and "metallic".
